doc/OutlierFiltersFamilies.md

```diff
@@ -0,0 +1,377 @@
+
+#### Andrew <a name="andrewhead"></a>
+
+Equations:
+![eq andrew](images/eq_andrew.JPG)
+
+Plots:
+![andrew threek](images/andrew_threek.png)
+![andrew multik](images/andrew_multik.png)
+
+Original reference:
+R.E. Welsch, "Robust Regression Using Iteratively Reweighted Least-squares", Communications in Statistics - Theory and Methods, vol. 6, no. 9, pp.813-827, 1977.
+
+#### Kernel-Correlation (KC) <a name="kernelcorrelationhead"></a>
+
+Kernel Correlation is a registration algorithm known to use the Welsch outlier filter.
+
+Equations:
+![eq kc](images/eq_kc.JPG)
+
+Original reference:
+Y.Tsin and T.Kanade, "A Correlation-Based Approach to Robust Point Set Registration", in 8th European Conference on Computer Vision, vol.0 , 2004, pp.558-569
+
+#### Fast Global Registration (FGR) <a name="fastglobalregistrationhead"></a>
+
+This registration algorithm is known to use the Geman-McClure outlier filter.
+
+Equations:
+![eq fgr](images/eq_fgr.JPG)
+
+Original reference:
+Q.Y. Zhou, J. Park, and V.Koltun, "Fast global registration", Lecture notes in Computer Science, vol.9906 LNCS, pp. 766-782,2016
+
+### Linear Family <a name="linearfamilyhead"></a>
+
+This family of outlier filters is characterized by an influence function that looks like a linear function. Appart from L2, the value of k affects the width of the linear part. Past the linear behavior, the influence function becomes null. Usually, inliers are affected by the linear part of the function while the outliers are affected by the null part of the function. The Switchable-Constraint outlier filter provides a smooth transition between the inlier and outlier part while Maximum Distance provides a sharp transition between those two behaviors. L2 represents the ideal linear influence function. On the plot Maximum Distance and GICP are overlapping.
+
+![linear Family plot](images/linear_family.png)
+
+#### L2 <a name="l2head"></a>
+
+L2 is a function which is a part of the Lp space family of functions. The L2 outlier filter can't be tuned with a k parameter.
+
+Equations:
+![eq l2](images/eq_l2.JPG)
+
+Plot:
+![L2 nok](images/L2_nok.png)
+
+#### Switchable-Constraint <a name="switchableconstrainthead"></a>
+
+Equations:
+![eq sc](images/eq_sc.JPG)
+
+Plots:
+![switchableconstraint threek](images/switchableconstraint_threek.png)
+![switchableconstraint multik](images/switchableconstraint_multik.png)
+
+Original reference:
+N. Süderhauf and P. Protzel,"Switchable constraint for robust pose graph slam", in 2012 IEEE/RSJ International Conference on Intelligent Robots and Systems, 2012, pp. 1879-1884. & P Argawal, G.D. Tipaldi, L. Spinello, C.Stachniss, and W.Burgard, "Robust map optimization using dynamic covariance scaling", IEEE international Conference on Robotics and Automation, pp. 62-69, 2013.
+
+#### Maximum Distance (MaxDist) <a name="maximumdistancehead"></a>
+
+The Maximum Distance outlier filter is used to compute the unscaled distance (e': euclidean error).
```

Comment:
This is your first mention of the unscaled distance. The scaled distance is the unscaled distance divided by the variance ([See this](https://arxiv.org/pdf/1810.01474.pdf) chapter III). In my thesis, I made the distinction between scale and unscaled distance, because for M-estimator there is an optimal value of k for a scaled distance. Basically, there is optimal value of k for set of inliers which have a Gaussian perturbation with a zero mean and a variance of 1 (for Huber it is k=1.345) . If we assume that our inliers will have Gaussian perturbation with a zero mean and an unknown variance, we need to be able estimate that variance from our residual. This unknown variance is the scale and we need a scale estimator (also called S-estimator). So if you take  In libpointmatcher, we have only a single scale estimator implemented: Median of absolute deviation (MAD). So if the user activates this scale estimator and is using a M-estimator as outlier filter, they won't need to fine tune k in theory.

Reply:
GICP deleted so I added a small version of your explanation in the paragraph "Effect of tuning parameter k"